### PR TITLE
add(library): Flyon UI

### PR DESCRIPTION
**Pull request by abhidave001**
| Name                 | Link                 |
| -------------------- | -------------------- |
| _Flyon UI_ | _https://flyonui.com/_ |

[FlyonUI](https://flyonui.com/) is designed to combine the best of both worlds: the aesthetic appeal of semantic classes and the powerful functionality of JS plugins.

It has over 78+ components & examples & built with DaisyUI & Preline.

**Features:**

- 800+ Free Components & Examples
- Universal Framework Compatibility
- Unlimited Themes 
- Unstyled & Accessible Plugins
- Responsive & RTL support
- Free Forever
- Beautiful and Semantic Styling
- Efficiency and Productivity
- Maintainable and Scalable

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by innocenzi** [comment]
Hello, please group it properly 👍

File: README.md [hunk 1]
```diff
@@ -243,6 +243,7 @@
 - 📁 [Astro Template Cactus](https://github.com/chrismwilliams/astro-theme-cactus) - Tailwind CSS Astro starter template.
 - 📁 [Astro Template Ovidius](https://github.com/JustGoodUI/ovidius-astro-theme) - Tailwind CSS & Astro blog template.
 - 📁 [Astro Template Dante](https://github.com/JustGoodUI/dante-astro-theme) - Tailwind CSS & Astro blog/portfolio template.
+- 📚🧩[Flyon UI Tailwind Components Library](https://flyonui.com/) - The Easiest Components Library For Tailwind CSS.
```
Comment (joshmanders) [inline README.md:246]:
```suggestion
- 📚🧩[Flyon UI](https://flyonui.com/) - Component Library For Tailwind CSS.
```

**Review by joshmanders — CHANGES_REQUESTED** [review]

**Comment by abhidave001** [comment]
> Hello, please group it properly 👍

Hi,
May I know how to group it?

I mean I have added the product in the section: UI Libraries, Components & Templates
Should it be mentioned after Preline?

image

File: README.md [hunk 2]
```diff
@@ -187,6 +187,7 @@
 - 📚 [TWC](https://react-twc.vercel.app) - Lightweight library to create reusable React + Tailwind CSS components in one line.
 - 📚 [Tremor](https://tremor.so) - React library to build charts and dashboards with Tailwind CSS.
 - 📚 [Preline UI](https://preline.co) - Open-source Tailwind CSS components library for any needs.
+- 📚🧩[Flyon UI Tailwind Components Library](https://flyonui.com/) - Open Source Components Library For Tailwind CSS.
```
Comment (innocenzi) [inline README.md:190]:
```suggestion
- 📚🧩[Flyon UI](https://flyonui.com/) - Open-source components library for Tailwind CSS.
```

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
> as suggested by the authorities 

lol—you checked "I have read and followed the contribution guidelines", but still didn't:
- Place your item correctly
- Name your pull request correctly
- Name your item correctly
- Name the description correctly

We just want to keep the list clean, man. Thanks for the contribution 🫠

**Comment by abhidave001** [comment]
> > as suggested by the authorities
> 
> lol—you checked "I have read and followed the contribution guidelines", but still didn't:
> 
> * Place your item correctly
> * Name your pull request correctly
> * Name your item correctly
> * Name the description correctly
> 
> We just want to keep the list clean, man. Thanks for the contribution 🫠
> 
> @joshmanders I think GitHub needs your re-approval before we can merge

Pardon me for the mistakes.

I did actually read the guidelines but thing is I wasn't sure about the title as mostly I have written short desc. of what I have added in the list whenever I submit the product. 

This is the case of misunderstanding from my end. 
I'll take care from now onwards.

Btw, Do I need to do anything from my end again?
Let me know.

**Comment by innocenzi** [comment]
All good—just missing Josh's approval, due to the previous request changes

**Review by joshmanders — APPROVED** [review]
> as suggested by the authorities

BOW TO ME, PEASANTS FOR I AM THE COMMANDER OF ALL AUTHORITY!

Just kidding, if Enzo says it's good, I think it's good to. Welcome to the list.

**Comment by abhidave001** [comment]
> > as suggested by the authorities
> 
> BOW TO ME, PEASANTS FOR I AM THE COMMANDER OF ALL AUTHORITY!
> 
> Just kidding, if Enzo says it's good, I think it's good to. Welcome to the list.

Thanks a lot for adding the resource. 👍🏻

**Pull request merged**